lib/action_subscriber/bunny/subscriber.rb

```diff
@@ -17,6 +17,9 @@ def auto_pop!
         times_to_pop = [::ActionSubscriber::Threadpool.ready_size, ::ActionSubscriber.config.times_to_pop].min
         times_to_pop.times do
           queues.each do |route, queue|
+            # Handle busy checks on a per threadpool basis
+            next if route.threadpool.busy_size >= route.threadpool.pool_size
```

Comment:
This is fine, but it could be easily rolled into the threadpool as a method called `busy?`:

```ruby
next if route.threadpool.busy?
```